velox/common/memory/SharedArbitrator.cpp

```diff
@@ -427,6 +427,8 @@ bool SharedArbitrator::arbitrateMemory(
 
   VELOX_CHECK_LT(freedBytes, growTarget);
   RECORD_METRIC_VALUE(kMetricArbitratorGlobalArbitrationCount);
```

Comment:
Maybe put the two steps in a utility say incrementGlobalArbitrationStats()? The same for localArbitrationStats? Thanks!